docs/cookbook.rst

```diff
@@ -35,6 +35,7 @@ documentation issue on `our GitHub repository <https://github.com/esm-tools/esm_
 .. } (Move to the end of the paragraph)
 .. : sort (sort the lines you have selected alphabetically)
 
-.. include:: recipes/modify_namelists.rst
 .. include:: recipes/add_model_setup.rst
+.. include:: recipes/echam_disturbance.rst
+.. include:: recipes/modify_namelists.rst
 .. include:: recipes/use_own_namelist.rst
```

Comment:
The names of the rst files do not correspond with the titles, and therefore, ordering them alphabetically by file name results in no order in the rendered version. I would suggest the following changes:
```
.. include:: recipes/echam_disturbance.rst       # Applying a Temporary...
.. include:: recipes/modify_namelists.rst           # Changing Namelist...
.. include:: recipes/add_model_setup.rst          # Implement a New...
.. include:: recipes/use_own_namelist.rst         # Using Your Own...
```

Reply:
Reorganized based upon actual title